Source/Analytics/Events.Admin/HealthRisks/HealthRiskCreated.cs

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
```

Comment:
For consistency with my other suggestions:
```suggestion

using System;
```